x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/auto_detect/auto_detect_panel.tsx

```diff
@@ -0,0 +1,239 @@
+                    title={i18n.translate(
+                      'xpack.observability_onboarding.autoDetectPanel.installingElasticAgentFlexItemLabel',
+                      { defaultMessage: 'Waiting for data to arrive...' }
+                    )}
+                  />
+                ) : status === 'inProgress' ? (
+                  <ProgressIndicator
+                    title={i18n.translate(
+                      'xpack.observability_onboarding.autoDetectPanel.lookingForLogFilesFlexItemLabel',
+                      { defaultMessage: 'Waiting for installation to complete...' }
+                    )}
+                  />
+                ) : null}
+                {(status === 'awaitingData' || status === 'dataReceived') &&
+                installedIntegrations.length > 0 ? (
+                  <>
+                    <EuiSpacer />
+                    {registryIntegrations.map((integration) => (
+                      <AccordionWithIcon
+                        key={integration.pkgName}
+                        id={`${accordionId}_${integration.pkgName}`}
+                        iconType="desktop"
+                        title={i18n.translate(
+                          'xpack.observability_onboarding.autoDetectPanel.h3.getStartedWithNginxLabel',
+                          {
+                            defaultMessage: 'Get started with {pkgName} logs',
+                            values: { pkgName: integration.pkgName },
+                          }
+                        )}
+                        isDisabled={status !== 'dataReceived'}
+                        initialIsOpen
+                      >
+                        <EuiFlexGroup responsive={false}>
+                          <EuiFlexItem grow={false}>
+                            {status === 'dataReceived' ? (
+                              <EuiImage
+                                src={http.staticAssets.getPluginAssetHref('charts_screen.svg')}
+                                width={162}
+                                height={117}
+                                alt=""
+                                hasShadow
+                              />
+                            ) : (
+                              <EuiSkeletonRectangle width={162} height={117} />
+                            )}
+                          </EuiFlexItem>
+                          <EuiFlexItem>
+                            <ul>
+                              {integration.kibanaAssets
+                                .filter((asset) => asset.type === 'dashboard')
+                                .map((dashboard) => (
+                                  <li key={dashboard.id}>
+                                    <LocatorButtonEmpty<DashboardLocatorParams>
+                                      locator={DASHBOARD_APP_LOCATOR}
+                                      params={{ dashboardId: dashboard.id }}
+                                      target="_blank"
```

Comment:
I think it's great opening in a new tab! However, when using it I didn't trust it because there's no indication this will happen, should we append a suitable icon to the end?

Reply:
Adds some visual noise as well though, so not 100% sure

Reply:
Unfortunately, buttons can only have one icon. It think it would be too confusing otherwise.

We could change the icon to an external link icon on the right hand side but then we would have to remove the current icons

Reply:
Let's discuss with @sileschristian once he's back - no strong opinion on it

Reply:
@thomheymann 

> We could change the icon to an external link icon on the right hand side but then we would have to remove the current icons

This makes sense. Let's do this. Dashboard icon is not bringing that much of value. Also, we are missing a [copy](https://www.figma.com/design/CPhMyRNOgo0wsEiaIMZJ14/Onboarding-Quick-Starts?node-id=1196-38068&t=6oGnHosLzzIK6cnp-4) that we usually we had before the link that gives better context.

![image](https://github.com/elastic/kibana/assets/127108526/f6b66768-bc4c-4f51-9ddc-bacf2230873a)